src/main/java/net/whistlingfish/harmony/protocol/OAReplyParser.java

```diff
@@ -24,7 +32,8 @@
         for (String pair : contents.split(":")) {
             Matcher matcher = kvRE.matcher(pair);
             if (!matcher.matches()) {
-                throw new AuthFailedException(format("failed to parse element in auth response: %s", pair));
+            	continue;
+               // throw new AuthFailedException(format("failed to parse element in auth response: %s", pair));
```

Comment:
On newer hubs some responses have no json data, I will remove the comment if this logic is ok